test/files/run/t5717.scala

```diff
@@ -13,9 +13,19 @@ object Test extends StoreReporterDirectTest {
   // Let the compiler tell us output dir
   // val sc = newCompiler("-cp", classpath, "-d", testOutput.path)
   // val out = sc.settings.outputDirs.getSingleOutput.get
+  // TODO check the reported errors
```

Comment:
for this, just rebase this PR on current 2.12.x, https://github.com/scala/scala/pull/5835 is merged